openapi/components/schemas/calendar/eventCommonFields.yaml

```diff
@@ -0,0 +1,74 @@
+type: object
+properties:
+  userId:
+    $ref: ../../schemas/id.yaml
+  authorId:
+    $ref: ../../schemas/id.yaml
+  allDay:
+    type: integer
+    example: 0
+    description: Whether the event is an all day event.
+    nullable: false
+  efficacy:
+    type: string
+    enum:
+      - self
+      - series
+    example: self
+    description:
+      Whether the requested change should apply only to the event with the given 'id', or to all events in the
+      series of which event 'id' is a member.
+  title:
+    type: string
+    example: The Yalta Conference
+    description: Title of event.
+  startDate:
+    type: string
+    example: 2024-03-05T19:00+10:00
```

Comment:
These dates should contain seconds as well: I was attempting to create one of these events via a POST based on this document, and couldn't figure out why I could not.

Reply:
Fixed (in schoolbox) by adding RFC3339_EXTENDED to the list of allowed date times.

Reply:
I'm not sure that will work: the issue is with this example, not with Schoolbox's date parsing.

This example start date doesn't contain a seconds component, and thus a date object can't be created from it from the RFC3339 format.

Reply:
@armatronic this is the change I made:
https://github.com/alaress/schoolbox/pull/30072/commits/22c2a64dd7de13761a10a1df58966c2a03c36ddd

Previously a validation check required that the dates be in RFC3339 format.  After the change both RFC339 and RFC3339_EXTENDED are accepted.  I have tested and both date formats now work when creating and updating events.

Reply:
@tomcdonnell EXTENDED will allow fractions of a second.

 2024-04-15T12:30:45.123456789Z

The example is missing the seconds segment entirely which is usually invalid unless parsing from a template for some reason.

I'd just add a `:00` as a suffix to the date example right before the time zone offset.

Apologies if unwarranted...did similar in past and curious..

Reply:
Right you are @Brahmah.  Thanks, now fixed.